app/src/androidTest/java/com/github/gogetters/letsgo/activities/ProfileActivityTest.kt

```diff
@@ -136,6 +136,24 @@ class ProfileActivityTest : EmulatedFirebaseTest() {
         // TODO Check that the actual fields are written to UI
     }
 
+    @Test
+    fun testBackBringsToMainActivity() {
```

Comment:
Merde I also did that...

Reply:
Ahah. Yea trying to squeze out extra % of coverage